tests/integration/acl.editposts.feature

```diff
@@ -67,7 +67,6 @@ Feature: Access Control when modifying posts
     @rolesEnabled
     Scenario: Anonymous users can not edit posts
         Given that I want to update a "Post"
```

Comment:
I think that since there is no "Authorization: Bearer xxxxxxx" header coming in, this test should expect a 401 error, not 403. Unless there's a different reason to expect 403 here ?